spec/requests/api/v1/business_intelligence/items_bi_request_spec.rb

```diff
@@ -2,29 +2,33 @@
 
 describe "Items BI API" do
   describe "Item class" do
-    it '.most_revenue' do
```

Comment:
Looks like the tests for this method were overwritten?

Reply:
Good catch. I had started working on this test right before needing to go to my check in and thought that I had copied most_revenue from somewhere else. Just added it back in.